src/modules/dashboard/components/ActionsPageFeed/ActionsPageEvent/ActionsPageEvent.tsx

```diff
@@ -427,39 +429,45 @@ const ActionsPageEvent = ({
             />
           </div>
           <div className={styles.details}>
-            <div className={styles.roles}>
-              {eventName &&
-                EVENT_ROLES_MAP[eventName] &&
-                EVENT_ROLES_MAP[eventName].map((role, index) => {
+            {eventName && EVENT_ROLES_MAP[eventName] && (
+              <div className={styles.roles}>
+                {EVENT_ROLES_MAP[eventName].map((role, index) => {
```

Comment:
Maybe write this as 
```suggestion
                {EVENT_ROLES_MAP[eventName]?.map((role, index) => {
```

It shouldn't be really necessary here, but it's a good practice to safeguard